gradio/pipelines_utils.py

```diff
@@ -482,3 +482,321 @@ def is_diffusers_pipeline_type(pipeline, class_name: str):
             "postprocess": lambda r: r["images"][0],
         }
     raise ValueError(f"Unsupported diffusers pipeline type: {type(pipeline)}")
+
+
+def handle_transformers_js_pipeline(pipeline: Any) -> Dict[str, Any]:
+    try:
+        from transformers_js_py import as_url, read_audio  # type: ignore
+    except ImportError as ie:
+        raise ImportError(
+            "transformers_js_py not installed. Please add `transformers_js_py` to the requirements option of your Gradio-Lite app"
+        ) from ie
+
+    if pipeline.task == "audio-classification":
+        return {
+            "inputs": [
+                components.Audio(
+                    type="filepath",
+                    label="Input",
+                    render=False,
+                ),
+            ],
+            "outputs": components.Label(label="Class", render=False),
+            "preprocess": lambda i: (
+                read_audio(
+                    i, pipeline.processor.feature_extractor.config["sampling_rate"]
+                ),
+            ),
+            "postprocess": lambda r: {i["label"]: i["score"] for i in r},
+        }
+    if pipeline.task == "automatic-speech-recognition":
+        return {
+            "inputs": [
+                components.Audio(
+                    type="filepath",
+                    label="Input",
+                    render=False,
+                ),
+            ],
+            "outputs": components.Textbox(label="Output", render=False),
+            "preprocess": lambda i: (
+                read_audio(
+                    i, pipeline.processor.feature_extractor.config["sampling_rate"]
+                ),
+            ),
+            "postprocess": lambda r: r["text"],
+        }
+    # if pipeline.task == "depth-estimation":
```

Comment:
Now there is no Gradio component that directly takes and displays the depth map as a 3D model, while we can do it by writing a some amount code like this: https://huggingface.co/spaces/whitphx/gradio-lite-transformers-js-depth-estimation/blob/c7f86e976ddea4d573e98e5c42ceff524da49fe8/index.html#L35-L113

Is this a good time to consider to support depth map somehow, e.g. extending `gr.Model3D` or creating `gr.DepthMap` (or more generally `gr.PointCloud`)? @abidlabs